micro_manager/adaptivity/adaptivity.py

```diff
@@ -107,3 +95,36 @@ def _check_for_deactivation(
                 if similarity_dists[active_id, active_id_2] < self._coarse_tol:
                     return True
         return False
+
+    def _get_similarity_measure(self, similarity_measure):
```

Comment:
I see that this functions is called once with a `similarity_measure` string and then with `data`. Just for my understanding, how does this work exactly?

Reply:
`_get_similarity_measure` returns a function which is saved as `self._similarity_measure`. This function is then called with `data` not `_get_similarity_measure`.
Or did you mean the similarity measure itself?